paddle/fluid/framework/selected_rows.cc

```diff
@@ -69,5 +116,66 @@ void DeserializeFromStream(std::istream& is, SelectedRows* selected_rows,
   TensorFromStream(is, selected_rows->mutable_value(), dev_ctx);
 }
 
+bool SelectedRows::HasKey(int64_t key) const {
```

Comment:
Maybe add some comments here to explain why we need these APIs for `SelectedRows` data structure.

Reply:
Done.